app/code/Magento/InventoryCatalog/view/adminhtml/web/template/product/stock-info.html

```diff
@@ -0,0 +1,21 @@
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<div class="admin__field-complex" data-bind="css: $data.additionalClasses, attr: {'data-index': index}">
```

Comment:
Use named attributes instead of `data-bind`. In this case:
```html
<div class="admin__field-complex" css="$data.additionalClasses" attr="'data-index': index">
```
Also add  `visible="visible"` attribute.

Reply:
I don't know why but with visible="visible" I got an empty Stock Information bar. 
<img width="1342" alt="screen shot 2018-01-29 at 10 14 45 pm" src="https://user-images.githubusercontent.com/10618641/35532268-51f3c9a6-0542-11e8-9c47-bcb6657c1937.png">

Reply:
You are right. You must also add visible item to configuration and then all must work fine.